diff --git a/lib/theme/SearchPage/index.js b/lib/theme/SearchPage/index.js
index 581f29799680b0dddf5fb9e4e07f5707a3cfbde0..ff6c1ca9cbd359fd13e8dfb44a4ebb9c34e95162 100644
--- a/lib/theme/SearchPage/index.js
+++ b/lib/theme/SearchPage/index.js
@@ -265,11 +265,7 @@ function SearchPageContent() {
           </div>
 
           <div className={clsx('col', 'col--4', 'text--right', styles.searchLogoColumn)}>
-            <a target="_blank" rel="noopener noreferrer" href={`https://typesense.org/?utm_medium=referral&utm_content=powered_by&utm_campaign=docsearch`} aria-label={translate({
-            id: 'theme.SearchPage.typesenseLabel',
-            message: 'Search by Typesense',
-            description: 'The ARIA label for Typesense mention',
-        })}>
+            <a target="_blank" rel="noopener noreferrer" href={`https://typesense.org/?utm_medium=referral&utm_content=powered_by&utm_campaign=docsearch`} aria-label={'Search by Typesense'}>
               <svg fill="none" height="21" viewBox="0 0 141 21" width="141" xmlns="http://www.w3.org/2000/svg">
                 <clipPath id="a">
                   <path d="m0 0h141v21h-141z"/>
diff --git a/src/theme/SearchPage/index.tsx b/src/theme/SearchPage/index.tsx
index f7f844cfff744447dac1fccf8dc5294eeca948da..5fba3284f8c43b527077a72ace3276aadb448df7 100644
--- a/src/theme/SearchPage/index.tsx
+++ b/src/theme/SearchPage/index.tsx
@@ -460,11 +460,7 @@ function SearchPageContent(): JSX.Element {
               target="_blank"
               rel="noopener noreferrer"
               href={`https://typesense.org/?utm_medium=referral&utm_content=powered_by&utm_campaign=docsearch`}
-              aria-label={translate({
-                id: 'theme.SearchPage.typesenseLabel',
-                message: 'Search by Typesense',
-                description: 'The ARIA label for Typesense mention',
-              })}>
+              aria-label={'Search by Typesense'}>
               <svg
                 fill="none"
                 height="21"